doc/examples/PullRequest.rst

```diff
@@ -43,3 +43,20 @@ Get Pull Requests by Query
     861
     875
     876
+    
+Add and modify Pull Request comment
+-----------------------------------
+
+.. code-block:: python
+
+    >>> repo = g.get_repo("PyGithub/PyGithub")
+    >>> pr = repo.get_pull(666)
+    >>> last_commit = pr.get_commits[pr.commits - 1]
```

Comment:
```suggestion
    >>> last_commit = pr.get_commits()[pr.commits - 1]
```
otherwise we see
```
TypeError: 'method' object is not subscriptable
```

Reply:
Fixed, thanks a lot